From e2b6e39f8cc45d88dd8107987f05f684e2283d39 Mon Sep 17 00:00:00 2001
From: Adrian Ratiu <adrian.ratiu@collabora.com>
Date: Thu, 25 Apr 2024 22:14:02 +0300
Subject: [PATCH] Cargo.toml: point to ebuild cursive dep

tbtools pulls the latest curive from github which we can't do,
so we fetch it via SRC_URI and point to the portage extracted
archive.

Upstream-Status: Not applicable [ebuild change]
Signed-off-by: Adrian Ratiu <adrian.ratiu@collabora.com>
---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index c17bd5a..506f4c8 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -6,7 +6,7 @@ edition = "2021"
 [dependencies]
 ansi_term = "0.12"
 clap = { version = "4.0", features = ["derive"] }
-cursive = { version = "0.20.0", default-features = false, features = ["crossterm-backend"] }
+cursive = { path = "../cursive-cursive_core-v0.3.7/cursive", default-features = false, features = ["crossterm-backend"] }
 is-terminal = "0.4.6"
 lazy_static = "1.4.0"
 nix = "0.26.2"
-- 
2.43.2

